### PR TITLE
Add Welcome Page

### PR DESCRIPTION
**Pull request by AnkePfei2fer**
- import images to display them correctly in deployment
- add welcome page to storybook

https://steinchen-welcomepage-obalufd2.herokuapp.com/storybook/?path=/story/pages-welcome--default

image

### CODE REVIEW

File: src/app/pages/LoginPage/loginPage.tsx [hunk 1]
```diff
@@ -16,16 +18,10 @@ export default function LoginPage() {
         <span className={styles.N}>N</span>
       </div>
       <h3 className={styles.enterName}>Gib Deinen Namen ein</h3>
-      <img
-        className={styles.minifig}
-        src="/../../../src/assets/images/Figur.png"
-      />
+      <img className={styles.minifig} src={FigurSrc} />
```
Comment (marwinburesch) [inline src/app/pages/LoginPage/loginPage.tsx]: `alt` is missing 🧐

File: src/app/pages/WelcomePage/welcomePage.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+              fill="#8C0327"
+            />
+          </svg>
```
Comment (marwinburesch) [inline src/app/pages/WelcomePage/welcomePage.tsx:36]: I think these SVG's could be components. Like this:
https://github.com/marwinburesch/moviescout/blob/main/src/app/components/assets/BookmarkIcon.tsx

File: src/app/pages/LoginPage/loginPage.module.css [hunk 3]
```diff
@@ -85,8 +85,9 @@
 
 .bricks {
   position: absolute;
-  bottom: 2px;
-  left: 5px;
+  width: 12rem;
+  bottom: 5px;
+  left: 10px;
```
Comment (marwinburesch) [inline src/app/pages/LoginPage/loginPage.module.css]: maybe use `em` here?

**Review by marwinburesch — COMMENTED** [review]
Nice

File: src/app/pages/WelcomePage/welcomePage.tsx (hunk 2 above)
Comment (AnkePfei2fer) [reply src/app/pages/WelcomePage/welcomePage.tsx:36]: Saved as Issue for later

**Pull request merged**